scipy/stats/_mannwhitneyu.py

```diff
@@ -304,6 +321,10 @@ def mannwhitneyu(x, y, use_continuity=True, alternative="two-sided",
           is less than or equal to 8 and there are no ties;
           chooses ``'asymptotic'`` otherwise.
 
+        `method` may also be an instance of `stats.PermutationMethod`, in which
```

Comment:
```suggestion
        * `scipy.stats.PermutationMethod`: `method` may also be an instance of `scipy.stats.PermutationMethod`, in which
```
I think Permutation method instance should be one of bullet item and making a doc link of `scipy.stats.PermutationMethod` is useful for readers. 
![IMG_1491](https://github.com/scipy/scipy/assets/3813847/1f37f428-bc97-4503-bb75-a62dceed4ac2)